dockerfiles/Dockerfile.browser

```diff
@@ -45,7 +45,8 @@ RUN find . -name package.json -type f -mindepth 2 -maxdepth 3 ! -path "./node_mo
 RUN rm -fr **/*/node_modules/
 
 # The maze-runner browser tests
-FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:latest-v3-cli as browser-maze-runner
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:tms-browser-corrections-cli as browser-maze-runner
+#FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:latest-v4-cli as browser-maze-runner
```

Comment:
Some corrections to MazeRunner were needed to support this change and are awaiting review. Once merged, MR can be re-released and this simply changed back.